controllers/authcontroller.js

```diff
@@ -0,0 +1,116 @@
+import User from '../models/user_schema';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcryptjs';
+import {promisify} from 'util';
+
+const signinToken= params=>{
+    return jwt.sign(params,'jsonWebToken_Password_Webtoken_Secret' /*,{
+        expiresIn:'60d'
+}*/);
+};
+
+exports.signUp=async(req,res)=>{
+   try{
+    const newUser=await User.create({
+        name:req.body.name,
+        email:req.body.email,
+        password:req.body.password,
+        passwordConfirm:req.body.passwordConfirm
+    });
+    const token=signinToken(newUser._id);
+    res.status(201).json({
+        status:'success',
+        token,
+        data:{
+            user:newUser
+        }
+    })
+   }
+   catch(err){
+    res.status(404).json({
+        status:'fail',
+        message:err
```

Comment:
Don't send errors to Postman, send a descriptive message that an error occurred then, handle the error in your code.